CHANGELOG.md

```diff
@@ -2,6 +2,70 @@
 
 <!-- next version -->
 
+## 0.99.0
+
+### 🛑 Breaking changes 🛑
+
+- `operator`: change java instrumentation feature gate operator.autoinstrumentation.java into command line flag --enable-java-instrumentation (#2673, #2582)
```

Comment:
Would be nice if this entry followed the format of the other two, and if they were grouped together.